base/expr.jl

```diff
@@ -61,6 +61,16 @@ Takes the expression `x` and returns an equivalent expression with all macros re
 """
 macroexpand(x::ANY) = ccall(:jl_macroexpand, Any, (Any,), x)
 
+"""
+    @macroexpand
+
+Return equivalent expression with all macros removed (expanded).
```

Comment:
Can you give a usecase this would actually make a difference? And why is `@macroexpand` potentially better at the repl?